messenger.rst

```diff
@@ -273,6 +273,116 @@ you can disable them like this:
                 messenger.bus.default:
                     default_middleware: false
 
+Using Middleware Factories
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes middleware are configurable using factories. There are two types of
```

Comment:
Turned like this, it seems to me the sentence seems is saying say there are two different ways to achieve configurable middleware, while it's actually two required services definitions. One registering the factory class as a service. The second one registering an abstract definition building the service through the first factory.

Reply:
I'm sorry for having changed the meaning of the docs. I've reworded it again to follow your explanation. Cheers!